### ISSUE TITLE
UX | Auth experience | Identify tools that make sense to add to auth menu

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#71332]
## Background
We recently had success adding dependents and letters to the authenticated menu. How might we apply that to surfacing other top authenticated tools for veterans?  Let's identify 3-5 tools as a starting point for conversation.

## Tasks

- [x] Review analytics and logged-in task research to identify 3-5 most used tools that require authentication on the site.  Might make sense to correlate that to search terms to narrow down the list and see which would be most useful to add to the menu
- [ ] Review with Samara

## Acceptance criteria

- [ ] We've decided whether or not we should move forward with this idea
- [ ] If we move forward, we know which tools we'll highlight in the auth menu

**Comment by trevor2718** [comment]
FUCK

**Comment by trevor2718** [comment]
I cannot work!

**Comment by andaleliz** [comment]
@trevor2718 is there something I can help you with?

**Comment by trevor2718** [comment]
@andaleliz it appears someone has hacked my account, I apologize for the inconvenience but I am looking to resolve it.

**Comment by andaleliz** [comment]
@mtcA6 this is something that isn't specific to profile - another one of those "not sure where it fits" items.

**Comment by Samara-Strauss** [comment]
@mtcA6 We could create an epic, though it might change in scope. I would classify this broadly as iteration and exploring updates on the auth menu. It's not a tightly scoped initiative at this point.

Between My VA and Profile, I don't have strong feelings about who owns the auth menu code, and I would welcome collaboration from anyone on either team to contribute to this effort over time. If clear ownership matters, we can talk about this in one of our product syncs.

**Comment by andaleliz** [comment]
Hi @Samara-Strauss! I took a look at analytics and have some thoughts about how to move forward with this work.  

TL;DR: I think the claim status, view payment history, online appointment scheduling, and view disability rating tools are the best candidates.  

**My approach**

I focused on logged in users rather than all users because they’re the ones who would see the auth menu and are most likely using the tools on the pages if they bothered to sign in.  Since I had that custom segment applied, I had to look at data from the last 93 days, so the numbers are from Sept-Dec. I limited my view to the top 50 URLs in the report, because the numbers become so small across all the weird URL variations, but as I type this I realize I probably shouldn’t have done that because the total of all those could be important :thinking_face:   However, I think  it’s still OK for this initial take.


| Tool                                                         | Sept - Dec 2023 Pageviews by authenticated users |
|--------------------------------------------------------------|--------------------------------------------------|
| Claim status tool (`track-claims/your-claims` , and `track-claims/index.html`) | 13,158,190                                       |
|  View payment history (`/va-payment-history/index.html` and `/va-payment-history/payments/index.html`) | 2,814,412                                        |
| Online appointments tool (`/my-health/appointments/` and `/health-care/schedule-view-va-appointments/index.html`) | 2,352,648                                        |
| View disability rating (`disability/view-disability-rating/rating/`) | 1,989,305                                        |

Then, I did the same thing for search results, since these are indicative of people not finding what they're looking for via navigation. These terms are related to tools, and aren't already linked from the auth menu (I saw things like "direct deposit" and "dependents" but we're already addressing those)


| Terms                                                 | Related tool                  | Sept - Dec 2023 searches by authenticated users |
|-------------------------------------------------------|-------------------------------|-------------------------------------------------|
| `upload documents` and `upload evidence`              | Claim status tool             | 11,110                                          |
| `payment history`                                     | View payment history          | 8,967                                           |
| `disability rating`                                   | View disability rating        | 4,980                                           |
| `claim status`                                        | Claim status tool             | 4,827                                           |
| `appointments`                                        | Online appointment scheduling | 4,316                                           |
| `claims` (this could be related to travel claims too) | Claim status tool             | 4,190                                           |

**My recommendation**

I think adding the claim status tool and appointments makes the most sense.  I'm on the fence about view payment history and view disability rating, only because they're not exactly actionable. Particularly disability rating.  However, we know both are very important to veterans so I can easily be pushed to the "let's do it" side of the fence.

If we move forward with the experiment, I think we could track a couple of things as success metrics:

- engagement w/ the nav link
- interactions within the tools, somehow. Something to show that any increase in traffic is parallel to increase with tool engagement. I don’t think we’d want to see more traffic and pageviews without engagement also going up.

What do you think? I haven't given much thought to IA and design treatments yet, other than I do think we'd have to chunk out the menu more given that this would put us at 8-10 items (including sign out).

**Comment by andaleliz** [comment]
Hi @Samara-Strauss checking back with you on this - surely the notification got buried in your inbox :) We can talk about this in our 1:1 today if you're ready.

**Comment by Samara-Strauss** [comment]
To reiterate from our IRL convo:

- Let's definitely add Claims and see what happens.
- I'm also inclined to add disability rating and see what happens, though the argument that it's not an action oriented page is a valid one.

**Comment by Samara-Strauss** [comment]
Things we didn't get to talk about but should -- after adding more links, at what point do we do usability testing/user research? When do we think testing of this menu is warranted?

**Comment by andaleliz** [comment]
@Samara-Strauss thanks for capturing this here! I'll continue the next steps in a separate ticket.
